homeassistant/components/camera/__init__.py

```diff
@@ -169,6 +169,20 @@ class Image:
     content: bytes = attr.ib()
 
 
+@dataclass(frozen=True)
+class CameraCapabilities:
+    """Camera capabilities."""
+
+    frontend_stream_types: set[StreamType]
+
+    def to_frontend_dict(self) -> dict[str, Any]:
+        """Return a dict that can be used by the frontend."""
+
+        return {
+            "frontendStreamTypes": self.frontend_stream_types,
```

Comment:
No other data that we send to the frontend is using the consumers casing format. We just output our data in the Python's used snake casing format and the frontend (or any other consumer) can translate it to their casing.